serialization/README.md

```diff
@@ -86,3 +86,43 @@ After making changes to content types or content items in Content Hub ONE, you s
    ```bash
    npm run pull:additional-items
    ```
+
+## Starting with an empty Content Hub ONE instance
```

Comment:
Please move this section just after `## Connecting a Content Hub ONE instance to the CLI`.

Then, move the other sections as H3 under a H2 named `## Operations`